python/kwiver/vital/types/image_container.h

```diff
@@ -6,33 +6,46 @@
 #define KWIVER_VITAL_PYTHON_IMAGE_CONTAINER_H_
 
 #include <pybind11/pybind11.h>
-#include <vital/types/image_container.h>
 #include <python/kwiver/vital/types/image.h>
+#include <vital/types/image_container.h>
```

Comment:
Unfortunately, "applying suggestions on deleted lines is not supported", so I can't make this a suggestion. But:
```c++
#include <python/kwiver/vital/types/image.h>

#include <vital/types/image_container.h>

#include <pybind11/pybind11.h>
```

(Adding better support for include groups is somewhere on my to-do list for Uncrustify.)

Reply:
I changed this according to your suggestion manually.